docker.go

```diff
@@ -181,17 +188,40 @@ func (provider *DockerProvider) loadDockerConfig(dockerClient *docker.Client) *C
 	}
 
 	if _, err := toml.Decode(buffer.String(), configuration); err != nil {
-		log.Error("Error creating docker configuration", err)
+		log.Error("Error creating docker configuration ", err)
 		return nil
 	}
 	return configuration
 }
 
-func getHost(container docker.Container) string {
+func (provider *DockerProvider) getFrontendName(container docker.Container) string {
+	// Replace '.' with '-' in quoted keys because of this issue https://github.com/BurntSushi/toml/issues/78
+	return strings.Replace(provider.GetFrontendRule(container)+"-"+provider.GetFrontendValue(container), ".", "-", -1)
```

Comment:
Long line, it's a nit but I would tend to write :

```go
frontendName := fmt.Sprintf("%s-%s", provider.GetFrontendRule(container), provider.GetFrontendValue(container))
return strings.Replace(frontendName, ".", "-", -1)
```

Or even worse, with more variables xD.